### DISCUSSION TITLE
Use sprinkles in a variant

### DISCUSSION THREAD

**Discussion by nayaabkhan** [vanilla-extract-css/vanilla-extract#538] [Q&A] [answered]
Hi, thanks a lot for the libraries! I'm enjoying the experience.

I was trying to dive into Sprinkles and wondering if it is possible (yet) to use them with variants?

```ts
import { styleVariants } from '@vanilla-extract/css'
import { atoms } from '@themes/atoms.css' // <- exports sprinkles

export const fontVariants = styleVariants({
  heading: atoms({
    fontSize: 'base',
    fontWeight: 'bold'
  }),
  subheading: atoms({
    fontSize: 'small',
    color: 'text.muted',
  }),
})
```

This results in a TS error:
`Type 'string' is not assignable to type 'ComplexStyleRule'`

**Comment by nayaabkhan** [comment, marked as answer]
I removed the usage of `styleVariants` and I think it works as expected. **Wondering if this is non-idiomatic in any way?**

```ts
import { atoms } from '@themes/atoms.css'

export const fontVariants = {
  heading: atoms({
    fontSize: 'base',
    fontWeight: 'bold'
  }),
  subheading: atoms({
    fontSize: 'small',
    color: 'text.muted',
  }),
}
```

**Reply by nayaabkhan** [reply]
Marking it answered for the time being.

**Reply by mattcompiles** [reply]
This is completely valid usage. You'd only want the `styleVariants` here if you want to combine the sprinkles with custom styles.
